### DISCUSSION TITLE
1.4.8

### DISCUSSION THREAD

**Discussion by stnguyen90** [appwrite/appwrite#7030] [Show and tell]
### Notable changes

* Fix certificate emails and add support for variables in email template subject by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6495
* Bump console to version 3.2.5 by @stnguyen90 in https://github.com/appwrite/appwrite/pull/7027
* Bump utopia database and storage versions by @abnegate in https://github.com/appwrite/appwrite/pull/7002

### Bug fixes

* Fixes cookie headers not being passed properly by router by @Meldiron in https://github.com/appwrite/appwrite/pull/7024
* Fix permission problem in deletes worker by @abnegate in https://github.com/appwrite/appwrite/pull/7013

### Miscellaneous

* Improve error handling in the realtime service by @TorstenDittmann in https://github.com/appwrite/appwrite/pull/6998
* Update the error code for unsupported protocol by @vermakhushboo in https://github.com/appwrite/appwrite/pull/7006
* Improve CI tests by executing them in parallel by @christyjacob4 in https://github.com/appwrite/appwrite/pull/6198
* Update README.md to add links to orchestration tools by @krishvsoni in https://github.com/appwrite/appwrite/pull/7011
* Update gitpod setup to install instead of update dependencies by @TorstenDittmann in https://github.com/appwrite/appwrite/pull/6938
* Remove analytics from install script by @TorstenDittmann in https://github.com/appwrite/appwrite/pull/7017
* Improve database logging by @abnegate in https://github.com/appwrite/appwrite/pull/7003
* Add VCS tests by @vermakhushboo in https://github.com/appwrite/appwrite/pull/6894
* Improve error messages by @gewenyu99 in https://github.com/appwrite/appwrite/pull/6487
* Add command to delete orphaned projects by @shimonewman in https://github.com/appwrite/appwrite/pull/7015

## New Contributors
* @krishvsoni made their first contribution in https://github.com/appwrite/appwrite/pull/7011

**Full Changelog**: https://github.com/appwrite/appwrite/compare/1.4.7...1.4.8

<hr /><em>This discussion was created from the release <a href='https://github.com/appwrite/appwrite/releases/tag/1.4.8'>1.4.8</a>.</em>

**Comment by usamashafiqs** [comment]
@stnguyen90 How do I delete orphaned projects? Is there any documentation available?
Are these projects without an organization? When I run migration, old deleted projects are still shown on the console output during migration. I am using AppWrite from v0.13.0.

**Reply by gewenyu99** [reply]
From which version to which version did you run your migration?

**Reply by usamashafiqs** [reply]
v1.3.8 to v1.4.x, but after v0.13.0, when I upgrade to the major version, I need to run migration in order to upgrade to that version, and deleted projects are still shown on the console output during migration.
Right now, I am on v1.4.9
